chirp/drivers/ic9x_ll.py

```diff
@@ -145,7 +145,7 @@ def __str__(self):
 
     def send(self, pipe, verbose=False):
         """Send the frame to the radio via @pipe"""
-        if verbose or True:
+        if verbose:
```

Comment:
This is leftover debug. I often do this to force a condition, intending to remove it later :)